sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/base/BaseComposeFragment.kt

```diff
@@ -35,7 +36,7 @@ abstract class BaseComposeFragment : Fragment(), BaseComposeFragmentCallbacks {
         savedInstanceState: Bundle?
     ): View? {
         return ComposeView(requireContext()).apply {
-            setContent { composeScreen.invoke() }
+            setContent { Theme { composeScreen.invoke() } }
```

Comment:
Quick question, are we gonna leave the theme as `Theme` and not something like `ComposeSampleTheme`? Not sure what our convention will be for this 😄 Altho I suppose convention is still being written so no biggie 😄